Code/Framework/AzCore/Tests/Math/AabbTests.cpp

```diff
@@ -410,6 +410,20 @@ namespace UnitTest
         EXPECT_TRUE(aabb.GetMax().IsClose(transAabb.GetMax()));
     }
 
+    TEST(MATH_AabbTransform, GetTransformedAabbCorrectResult)
```

Comment:
probably can parameterize this to add more variations.

Reply:
Apologies for the delay in replying, was working on something else last week. 

I'd say there is value in going from 0 tests to 1, but I think going from 1 to many tests would have diminishing returns here, especially since there aren't multiple code paths in the implementation. I'm also a bit reluctant to add a  parameterized test because it would make it a bit harder to read/debug the test.